pkgs/servers/draupnir/test-build.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl common-updater-scripts nodePackages.node2nix gnused nix coreutils jq yarn yarn2nix
+
+if [ ! -z "$CLEAN" ]; then
+  nix-env -e draupnir
+  nix-store --gc
+  nix-collect-garbage -d
+
+fi
+
+nix-build -K -A draupnir
+
+if [ ! -z "$INSTALL" ]; then
+  nix-env -e draupnir
+  nix-env -f . -iA draupnir
+fi
+
+if [ ! -z "$RUN" ]; then
+  draupnir
+fi
```

Comment:
Adding a trailing newline would be helpful to make this a well-formed UNIX text file (which requires that each line of text be terminated with a newline, as opposed to merely using newlines to delimit lines).